src/containers/MoviesList/movieslist.js

```diff
@@ -13,17 +14,28 @@ const MoviesList = ({ onMovieClick }) => {
     return state.movies.status;
   });
   
+  const location = useLocation();
+  
+  const urlValue = location.search;
+  const searchValue = new URLSearchParams(urlValue).get('search');
+  
+  const category = useSelector((state) => selectCategory(state));
+  
+  //TODO: fix
```

Comment:
fix what? :)

Reply:
by some reason this useEffect executes 2 times, I don't know why...